obspy/taup/tau.py

```diff
@@ -663,3 +713,91 @@ def create_taup_model(model_name, output_dir, input_dir):
     else:
         model_file_name = model_name + ".tvel"
     TauPCreate.main(model_file_name, output_dir, input_dir)
+
+
+def traveltime_plot(min_degree=0, max_degree=360, npoints=1000,
+                    phases=['P', 'S'], source_depth=10, model='iasp91',
+                    ax=None, show=True):
+        """
+        Method to plot the travel times of arrivals class, if any have
+        been calculated.
+
+        :param min_degree: minimum distance from the source (in degrees) to 
+            plot travel times Defaults to ``0``.
+        :type min_degree: float, optional
+        :param max_degree: maximum distance from the source (in degrees) to
+            plot travel times. Defaults to ``360``.
+        :type max_degree: float, optional
+        :param npoints: Number of points to plot. Defaults to ``1000``.
+        :type npoints: int, optional
+        :param phases: List of phase names which should be used within the 
+            plot. Defaults to ``['P', 'S']``.
+        :type phases: list of str, optional
+        :param source_depth: Source pepth in kilometers. Defaults to ``10``.
+        :type source_depth: float, optional
+        :param model: string containing the model to use.
+        :type model: str. Defaults to 'iasp91'
```

Comment:
The default value should be mentioned in the `:param ..:` section below, I think, rather than the `:type :` section